### PR TITLE
Resolving conflicts

### PR DESCRIPTION
**Pull request by goggi772**
Resolving conflicts

### CODE REVIEW

File: src/engine/Core.java [hunk 1]
```diff
@@ -234,162 +233,204 @@ public static void main(final String[] args) {
...
-					highScores = getFileManager().loadHighScores();
-					for (Score sc : highScores) {
-						if (sc.getScore() == 0 && sc.getAccuracy() == 0 && sc.getBullets() == 0 && sc.getKilled() == 0) {
-							result_stage = sc.getStage();
-							break;
-						}
+			try {
+				// load stage
+				highScores = getFileManager().loadHighScores();
+				for (Score sc : highScores) {
```
Comment (beom-i) [inline src/engine/Core.java:245]: Good change.

File: src/engine/Core.java [hunk 2]
```diff
@@ -234,162 +233,204 @@ public static void main(final String[] args) {
...
+							+ " title screen at " + FPS + " fps.");
+					returnCode = frame.setScreen(currentScreen);
+					LOGGER.info("Closing title screen.");
 					break;
-				}
 
-				// Show scenario pages
-				for(Scenario scenario : scenarios) {
-					ScenarioScreen scenarioScreen = new ScenarioScreen(width, height, FPS, scenario);
-					frame.setScreen(scenarioScreen);
```
Comment (beom-i) [inline src/engine/Core.java:294]: Good.

**Review by beom-i — APPROVED** [review]

**Comment by hungrypro7** [comment]
Thank you for your effors. Good!

File: src/engine/Core.java (hunk 1 above)
Comment (hungrypro7) [reply src/engine/Core.java:245]: Good!

**Pull request merged**